pkgs/applications/science/logic/nusmv/default.nix

```
@@ -0,0 +1,33 @@
+{ stdenv
+, lib
+, fetchurl
+, autoPatchelfHook
+}:
+
+stdenv.mkDerivation rec {
+  pname = "NuSMV";
+  version = "2.6.0";
+
+  src = fetchurl {
+    url = "https://nusmv.fbk.eu/distrib/NuSMV-${version}-linux64.tar.gz";
+    sha256 = "1370x2vwjndv9ham5q399nn84hvhm1gj1k7pq576qmh4pi12xc8i";
+  };
+
+
+  nativeBuildInputs = [ autoPatchelfHook ];
+
+  installPhase = ''
+    install -m755 -D bin/NuSMV $out/bin/NuSMV
+    install -m755 -D bin/ltl2smv $out/bin/ltl2smv
+    cp -r include $out/include
+    cp -r share $out/share
+  '';
+
+  meta = with lib; {
+    description = "A new symbolic model checker for the analysis of synchronous finite-state and infinite-state systems";
+    homepage = "https://nuxmv.fbk.eu/pmwiki.php";
+    maintainers = with maintainers; [ mgttlinger ];
+    sourceProvenance = with sourceTypes; [ binaryNativeCode ];
+    platforms = platforms.linux;
```

Last fixup: if the binary file you are packaging here is for Linux Intel 64 bits, then you should edit this line:

```suggestion
    platforms = [ "x86_64-linux" ];
```

---

I resolved it by implementing 32bit binary pulling as well.